include/bluetooth/conn.h

```diff
@@ -148,6 +148,8 @@ struct bt_conn_le_info {
 	u16_t interval; /** Connection interval */
 	u16_t latency; /** Connection slave latency */
 	u16_t timeout; /** Connection supervision timeout */
+
+	const u8_t *chan_map; /** Connection channel map. */
```

Comment:
Why can't we `#ifdef` this behind a `CONFIG_BT_CH_MAP` Kconfig option so that it doesn't take additional space?

Reply:
We can ifdef the feature out. But as far as as I know `#ifdef` are not used in public header files, so we would still have this one here. If the feature is compiled out it would always be NULL.